contrib/sbase/Makefile

```diff
@@ -0,0 +1,215 @@
+TOPDIR = $(realpath ../..)
+
+HDR =\
```

Comment:
In all `Makefile` files I remember we put extra space before backslash (purely cosmetic remark).